tests/testthat/test-find_dependencies.R

```diff
@@ -0,0 +1,41 @@
+test_that("outputs the expected tibble", {
```

Comment:
Reading the titles of the tests in this file should explain what behavior this function adds.